src/styles/block/calendar.scss

```
@@ -0,0 +1,41 @@
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gapsize;
+  max-width: $width-cal;
+  padding: 20px;
+
+  @each $day, $number in $daynumber {
+    &--start-day--#{$day} :nth-child(1) {
+      margin-left: ($size-char + $gapsize) * $number;
+    }
+  }
+
+  @for $indexDay from 28 through 31 {
+    &--month-length--#{$indexDay} :nth-child(n + #{$indexDay + 1}) {
+      display: none;
+    }
+  }
+
+  &__day {
+    box-sizing: border-box;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    @extend %font-sq;
+    @extend %square-size;
+    transition: color 0.2s, transform 0.5s;
+
```

```suggestion
  &__day {
    display: flex;
    align-items: center;
    justify-content: center;
    @extend %square-size;
    box-sizing: border-box;
    
    @extend %font-sq;
    
    transition: color 0.2s, transform 0.5s;
```

checklist point 12. .declaration-order